clash-prelude/src/Clash/Signal/Trace.hs

```diff
@@ -373,7 +372,10 @@ dumpVCD## (offset, cycles) traceMap now
  where
   offensiveNames = filter (any (not . printable)) traceNames
 
-  labels = map chr [33..126]
+  -- Generate labels like reversed digits: 0:[1,2,01,11,21,02,12,22,001,...]
```

Comment:
```suggestion
  -- Generate labels almost like digits: if the alphabet were "012", generate
  -- [0,1,2,00,01,02,10,11,12, ..]
```

Although frankly it can do without the comment; it's just that the current comment is wrong. Feel free to just omit it altogether.